src/main/java/com/ivanmostovyi/demo/config/AsyncConfig.java

```diff
@@ -24,6 +24,13 @@ public AsyncConfig(ApplicationContext context) {
         this.context = context;
     }
 
+    @Override
```

Comment:
I believe we can have only logger in this class. No need for any logic, because those cases are handled already.